tests/lib/prepare.sh

```diff
@@ -164,6 +164,11 @@ EOF
     fi
 
     if [[ "$SPREAD_SYSTEM" == debian-* || "$SPREAD_SYSTEM" == ubuntu-* ]]; then
+        if [[ "$SPREAD_SYSTEM" == ubuntu-* ]]; then
+            apt install -y pollinate
```

Comment:
could you add a -q to that, and/or a `quiet` at the beginning? Without either of those, we'll have progress bars in the travis logs